admin/featured_categories.php

```diff
@@ -0,0 +1,554 @@
+                        break;
+
+                        default:
+                            if (isset($fInfo) && is_object($fInfo)) {
+                                $heading[] = ['text' => '<h4>ID#' . $fInfo->categories_id . ': ' . zen_clean_html($fInfo->categories_name) . '"</h4>'];
+                                $contents[] = [
+                                'align' => 'text-center',
+                                'text' => '
+                                <a href="' . zen_href_link(FILENAME_FEATURED_CATEGORIES, '&fID=' . $fInfo->featured_categories_id . '&action=edit' . ($currentPage != 0 ? '&page=' . $currentPage : '') . (isset($_GET['search']) ? '&search=' . $_GET['search'] : '')) . '" class="btn btn-primary" role="button">' . IMAGE_EDIT . '</a>
+                                <a href="' . zen_href_link(FILENAME_FEATURED_CATEGORIES, '&fID=' . $fInfo->featured_categories_id . '&action=delete' . ($currentPage != 0 ? '&page=' . $currentPage : '') . (isset($_GET['search']) ? '&search=' . $_GET['search'] : '')) . '" class="btn btn-warning" role="button">' . TEXT_INFO_HEADING_DELETE_FEATURED . '</a>'
+                                ];
+                                $contents[] = ['text' => TEXT_FEATURED_AVAILABLE_DATE . ' ' . (($fInfo->featured_date_available !== '0001-01-01' && $fInfo->featured_date_available !== '') ? zen_date_short($fInfo->featured_date_available) : TEXT_NONE)];
+                                $contents[] = ['text' => TEXT_FEATURED_EXPIRES_DATE . ' ' . (($fInfo->expires_date !== '0001-01-01' && $fInfo->expires_date !== '') ? zen_date_short($fInfo->expires_date) : TEXT_NONE)];
+                                if ($fInfo->date_status_change !== null && $fInfo->date_status_change !== '0001-01-01 00:00:00') {
+                                    $contents[] = ['text' => TEXT_INFO_STATUS_CHANGED . ' ' . zen_date_short($fInfo->date_status_change)];
+                                }
+                                $contents[] = ['text' => TEXT_INFO_LAST_MODIFIED . ' ' . zen_date_short($fInfo->featured_last_modified)];
+                                $contents[] = ['text' => TEXT_INFO_DATE_ADDED . ' ' . zen_date_short($fInfo->featured_date_added)];
+                                $contents[] = [
+                                'align' => 'text-center',
+                                'text' => zen_info_image($fInfo->categories_image, htmlspecialchars($fInfo->categories_name, ENT_COMPAT, CHARSET, true), SMALL_IMAGE_WIDTH, SMALL_IMAGE_HEIGHT)
+                                ];
+                                $contents[] = [
+                                'align' => 'text-center',
+                                'text' => '<a href="' . zen_href_link(FILENAME_CATEGORIES, '&cPath=&cID=' .$fInfo->categories_id . '&action=edit_category' ). '" class="btn btn-primary" role="button">' . IMAGE_EDIT_CATEGORY . '</a>'
```

Comment:
- Why the empty `&cPath` parameter?
- Add spaces around all concatenation operators.